src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import core.basesyntax.controller.BetConsoleHandler;
+import core.basesyntax.controller.GamblerConsoleHandler;
+
+public class Main {
+    public static void main(String[] args) {
+        BetConsoleHandler betConsoleHandler = new BetConsoleHandler();
+        betConsoleHandler.handle();
+        GamblerConsoleHandler gamblerConsoleHandler = new GamblerConsoleHandler();
+        gamblerConsoleHandler.handle();
```

Comment:
let’s have only one handler that will read info about both entities from the console
example of user input `200 0.6 Masha 20`

Reply:
done